scripts/lint-style.lean

```diff
@@ -109,10 +111,16 @@ def lintStyleCli (args : Cli.Parsed) : IO UInt32 := do
     | true => ErrorFormat.github
     | false => ErrorFormat.humanReadable
   let fix := args.hasFlag "fix"
+  if (args.variableArgsAs? String).isNone then
+    IO.println "error: invalid input, library arguments must be strings"
+    return 1
+  let libraries := match args.variableArgsAs! String with
+  | #[] => #["Archive", "Counterexamples", "Mathlib"]
+  | lib => lib
   -- Read all module names to lint.
   let mut allModuleNames := #[]
-  for s in ["Archive.lean", "Counterexamples.lean", "Mathlib.lean"] do
-    allModuleNames := allModuleNames.append (← findImports s)
+  for s in libraries do
+    allModuleNames := allModuleNames.append (← findImports s!"{s}.lean")
```

Comment:
I just noticed this: maybe using `System.FilePath.addExtension` is more robust?

Reply:
Good idea, fixed.